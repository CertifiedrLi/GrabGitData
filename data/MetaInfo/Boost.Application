{
  "id": 13566651,
  "name": "Boost.Application",
  "full_name": "retf/Boost.Application",
  "owner": {
    "login": "retf",
    "id": 504812,
    "avatar_url": "https://avatars.githubusercontent.com/u/504812?",
    "gravatar_id": "5ac4d7b9f6b85267c407016c21a91399",
    "url": "https://api.github.com/users/retf",
    "html_url": "https://github.com/retf",
    "followers_url": "https://api.github.com/users/retf/followers",
    "following_url": "https://api.github.com/users/retf/following{/other_user}",
    "gists_url": "https://api.github.com/users/retf/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/retf/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/retf/subscriptions",
    "organizations_url": "https://api.github.com/users/retf/orgs",
    "repos_url": "https://api.github.com/users/retf/repos",
    "events_url": "https://api.github.com/users/retf/events{/privacy}",
    "received_events_url": "https://api.github.com/users/retf/received_events",
    "type": "User",
    "site_admin": false
  },
  "private": false,
  "html_url": "https://github.com/retf/Boost.Application",
  "description": "Boost Application Library",
  "fork": false,
  "url": "https://api.github.com/repos/retf/Boost.Application",
  "forks_url": "https://api.github.com/repos/retf/Boost.Application/forks",
  "keys_url": "https://api.github.com/repos/retf/Boost.Application/keys{/key_id}",
  "collaborators_url": "https://api.github.com/repos/retf/Boost.Application/collaborators{/collaborator}",
  "teams_url": "https://api.github.com/repos/retf/Boost.Application/teams",
  "hooks_url": "https://api.github.com/repos/retf/Boost.Application/hooks",
  "issue_events_url": "https://api.github.com/repos/retf/Boost.Application/issues/events{/number}",
  "events_url": "https://api.github.com/repos/retf/Boost.Application/events",
  "assignees_url": "https://api.github.com/repos/retf/Boost.Application/assignees{/user}",
  "branches_url": "https://api.github.com/repos/retf/Boost.Application/branches{/branch}",
  "tags_url": "https://api.github.com/repos/retf/Boost.Application/tags",
  "blobs_url": "https://api.github.com/repos/retf/Boost.Application/git/blobs{/sha}",
  "git_tags_url": "https://api.github.com/repos/retf/Boost.Application/git/tags{/sha}",
  "git_refs_url": "https://api.github.com/repos/retf/Boost.Application/git/refs{/sha}",
  "trees_url": "https://api.github.com/repos/retf/Boost.Application/git/trees{/sha}",
  "statuses_url": "https://api.github.com/repos/retf/Boost.Application/statuses/{sha}",
  "languages_url": "https://api.github.com/repos/retf/Boost.Application/languages",
  "stargazers_url": "https://api.github.com/repos/retf/Boost.Application/stargazers",
  "contributors_url": "https://api.github.com/repos/retf/Boost.Application/contributors",
  "subscribers_url": "https://api.github.com/repos/retf/Boost.Application/subscribers",
  "subscription_url": "https://api.github.com/repos/retf/Boost.Application/subscription",
  "commits_url": "https://api.github.com/repos/retf/Boost.Application/commits{/sha}",
  "git_commits_url": "https://api.github.com/repos/retf/Boost.Application/git/commits{/sha}",
  "comments_url": "https://api.github.com/repos/retf/Boost.Application/comments{/number}",
  "issue_comment_url": "https://api.github.com/repos/retf/Boost.Application/issues/comments/{number}",
  "contents_url": "https://api.github.com/repos/retf/Boost.Application/contents/{+path}",
  "compare_url": "https://api.github.com/repos/retf/Boost.Application/compare/{base}...{head}",
  "merges_url": "https://api.github.com/repos/retf/Boost.Application/merges",
  "archive_url": "https://api.github.com/repos/retf/Boost.Application/{archive_format}{/ref}",
  "downloads_url": "https://api.github.com/repos/retf/Boost.Application/downloads",
  "issues_url": "https://api.github.com/repos/retf/Boost.Application/issues{/number}",
  "pulls_url": "https://api.github.com/repos/retf/Boost.Application/pulls{/number}",
  "milestones_url": "https://api.github.com/repos/retf/Boost.Application/milestones{/number}",
  "notifications_url": "https://api.github.com/repos/retf/Boost.Application/notifications{?since,all,participating}",
  "labels_url": "https://api.github.com/repos/retf/Boost.Application/labels{/name}",
  "releases_url": "https://api.github.com/repos/retf/Boost.Application/releases{/id}",
  "created_at": "2013-10-14T16:26:59Z",
  "updated_at": "2014-06-24T12:40:22Z",
  "pushed_at": "2014-06-24T12:40:22Z",
  "git_url": "git://github.com/retf/Boost.Application.git",
  "ssh_url": "git@github.com:retf/Boost.Application.git",
  "clone_url": "https://github.com/retf/Boost.Application.git",
  "svn_url": "https://github.com/retf/Boost.Application",
  "homepage": null,
  "size": 3009,
  "stargazers_count": 14,
  "watchers_count": 14,
  "language": "C++",
  "has_issues": true,
  "has_downloads": true,
  "has_wiki": true,
  "forks_count": 3,
  "mirror_url": null,
  "open_issues_count": 2,
  "forks": 3,
  "open_issues": 2,
  "watchers": 14,
  "default_branch": "master",
  "network_count": 3,
  "subscribers_count": 11
}
