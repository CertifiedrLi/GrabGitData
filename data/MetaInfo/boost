{
  "id": 296242,
  "name": "boost",
  "full_name": "jhairtt/boost",
  "owner": {
    "login": "jhairtt",
    "id": 91888,
    "avatar_url": "https://avatars.githubusercontent.com/u/91888?",
    "gravatar_id": "8c692e5e3cfab8cd6a45d035340f9442",
    "url": "https://api.github.com/users/jhairtt",
    "html_url": "https://github.com/jhairtt",
    "followers_url": "https://api.github.com/users/jhairtt/followers",
    "following_url": "https://api.github.com/users/jhairtt/following{/other_user}",
    "gists_url": "https://api.github.com/users/jhairtt/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/jhairtt/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/jhairtt/subscriptions",
    "organizations_url": "https://api.github.com/users/jhairtt/orgs",
    "repos_url": "https://api.github.com/users/jhairtt/repos",
    "events_url": "https://api.github.com/users/jhairtt/events{/privacy}",
    "received_events_url": "https://api.github.com/users/jhairtt/received_events",
    "type": "User",
    "site_admin": false
  },
  "private": false,
  "html_url": "https://github.com/jhairtt/boost",
  "description": "",
  "fork": false,
  "url": "https://api.github.com/repos/jhairtt/boost",
  "forks_url": "https://api.github.com/repos/jhairtt/boost/forks",
  "keys_url": "https://api.github.com/repos/jhairtt/boost/keys{/key_id}",
  "collaborators_url": "https://api.github.com/repos/jhairtt/boost/collaborators{/collaborator}",
  "teams_url": "https://api.github.com/repos/jhairtt/boost/teams",
  "hooks_url": "https://api.github.com/repos/jhairtt/boost/hooks",
  "issue_events_url": "https://api.github.com/repos/jhairtt/boost/issues/events{/number}",
  "events_url": "https://api.github.com/repos/jhairtt/boost/events",
  "assignees_url": "https://api.github.com/repos/jhairtt/boost/assignees{/user}",
  "branches_url": "https://api.github.com/repos/jhairtt/boost/branches{/branch}",
  "tags_url": "https://api.github.com/repos/jhairtt/boost/tags",
  "blobs_url": "https://api.github.com/repos/jhairtt/boost/git/blobs{/sha}",
  "git_tags_url": "https://api.github.com/repos/jhairtt/boost/git/tags{/sha}",
  "git_refs_url": "https://api.github.com/repos/jhairtt/boost/git/refs{/sha}",
  "trees_url": "https://api.github.com/repos/jhairtt/boost/git/trees{/sha}",
  "statuses_url": "https://api.github.com/repos/jhairtt/boost/statuses/{sha}",
  "languages_url": "https://api.github.com/repos/jhairtt/boost/languages",
  "stargazers_url": "https://api.github.com/repos/jhairtt/boost/stargazers",
  "contributors_url": "https://api.github.com/repos/jhairtt/boost/contributors",
  "subscribers_url": "https://api.github.com/repos/jhairtt/boost/subscribers",
  "subscription_url": "https://api.github.com/repos/jhairtt/boost/subscription",
  "commits_url": "https://api.github.com/repos/jhairtt/boost/commits{/sha}",
  "git_commits_url": "https://api.github.com/repos/jhairtt/boost/git/commits{/sha}",
  "comments_url": "https://api.github.com/repos/jhairtt/boost/comments{/number}",
  "issue_comment_url": "https://api.github.com/repos/jhairtt/boost/issues/comments/{number}",
  "contents_url": "https://api.github.com/repos/jhairtt/boost/contents/{+path}",
  "compare_url": "https://api.github.com/repos/jhairtt/boost/compare/{base}...{head}",
  "merges_url": "https://api.github.com/repos/jhairtt/boost/merges",
  "archive_url": "https://api.github.com/repos/jhairtt/boost/{archive_format}{/ref}",
  "downloads_url": "https://api.github.com/repos/jhairtt/boost/downloads",
  "issues_url": "https://api.github.com/repos/jhairtt/boost/issues{/number}",
  "pulls_url": "https://api.github.com/repos/jhairtt/boost/pulls{/number}",
  "milestones_url": "https://api.github.com/repos/jhairtt/boost/milestones{/number}",
  "notifications_url": "https://api.github.com/repos/jhairtt/boost/notifications{?since,all,participating}",
  "labels_url": "https://api.github.com/repos/jhairtt/boost/labels{/name}",
  "releases_url": "https://api.github.com/repos/jhairtt/boost/releases{/id}",
  "created_at": "2009-09-03T09:53:55Z",
  "updated_at": "2014-01-18T04:39:54Z",
  "pushed_at": "2009-09-03T09:56:28Z",
  "git_url": "git://github.com/jhairtt/boost.git",
  "ssh_url": "git@github.com:jhairtt/boost.git",
  "clone_url": "https://github.com/jhairtt/boost.git",
  "svn_url": "https://github.com/jhairtt/boost",
  "homepage": "",
  "size": 76,
  "stargazers_count": 1,
  "watchers_count": 1,
  "language": "C++",
  "has_issues": true,
  "has_downloads": true,
  "has_wiki": true,
  "forks_count": 0,
  "mirror_url": null,
  "open_issues_count": 0,
  "forks": 0,
  "open_issues": 0,
  "watchers": 1,
  "default_branch": "master",
  "network_count": 0,
  "subscribers_count": 1
}
