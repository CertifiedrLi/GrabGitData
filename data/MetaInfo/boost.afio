{
  "id": 10361921,
  "name": "boost.afio",
  "full_name": "BoostGSoC13/boost.afio",
  "owner": {
    "login": "BoostGSoC13",
    "id": 4558366,
    "avatar_url": "https://avatars.githubusercontent.com/u/4558366?",
    "gravatar_id": null,
    "url": "https://api.github.com/users/BoostGSoC13",
    "html_url": "https://github.com/BoostGSoC13",
    "followers_url": "https://api.github.com/users/BoostGSoC13/followers",
    "following_url": "https://api.github.com/users/BoostGSoC13/following{/other_user}",
    "gists_url": "https://api.github.com/users/BoostGSoC13/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/BoostGSoC13/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/BoostGSoC13/subscriptions",
    "organizations_url": "https://api.github.com/users/BoostGSoC13/orgs",
    "repos_url": "https://api.github.com/users/BoostGSoC13/repos",
    "events_url": "https://api.github.com/users/BoostGSoC13/events{/privacy}",
    "received_events_url": "https://api.github.com/users/BoostGSoC13/received_events",
    "type": "Organization",
    "site_admin": false
  },
  "private": false,
  "html_url": "https://github.com/BoostGSoC13/boost.afio",
  "description": "Proposed GSoC 2013 async file i/o library for Boost",
  "fork": false,
  "url": "https://api.github.com/repos/BoostGSoC13/boost.afio",
  "forks_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/forks",
  "keys_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/keys{/key_id}",
  "collaborators_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/collaborators{/collaborator}",
  "teams_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/teams",
  "hooks_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/hooks",
  "issue_events_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/issues/events{/number}",
  "events_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/events",
  "assignees_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/assignees{/user}",
  "branches_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/branches{/branch}",
  "tags_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/tags",
  "blobs_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/git/blobs{/sha}",
  "git_tags_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/git/tags{/sha}",
  "git_refs_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/git/refs{/sha}",
  "trees_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/git/trees{/sha}",
  "statuses_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/statuses/{sha}",
  "languages_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/languages",
  "stargazers_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/stargazers",
  "contributors_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/contributors",
  "subscribers_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/subscribers",
  "subscription_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/subscription",
  "commits_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/commits{/sha}",
  "git_commits_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/git/commits{/sha}",
  "comments_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/comments{/number}",
  "issue_comment_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/issues/comments/{number}",
  "contents_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/contents/{+path}",
  "compare_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/compare/{base}...{head}",
  "merges_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/merges",
  "archive_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/{archive_format}{/ref}",
  "downloads_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/downloads",
  "issues_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/issues{/number}",
  "pulls_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/pulls{/number}",
  "milestones_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/milestones{/number}",
  "notifications_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/notifications{?since,all,participating}",
  "labels_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/labels{/name}",
  "releases_url": "https://api.github.com/repos/BoostGSoC13/boost.afio/releases{/id}",
  "created_at": "2013-05-29T14:47:23Z",
  "updated_at": "2014-07-02T13:48:53Z",
  "pushed_at": "2014-05-26T16:49:12Z",
  "git_url": "git://github.com/BoostGSoC13/boost.afio.git",
  "ssh_url": "git@github.com:BoostGSoC13/boost.afio.git",
  "clone_url": "https://github.com/BoostGSoC13/boost.afio.git",
  "svn_url": "https://github.com/BoostGSoC13/boost.afio",
  "homepage": null,
  "size": 8569,
  "stargazers_count": 16,
  "watchers_count": 16,
  "language": "C++",
  "has_issues": true,
  "has_downloads": true,
  "has_wiki": true,
  "forks_count": 5,
  "mirror_url": null,
  "open_issues_count": 1,
  "forks": 5,
  "open_issues": 1,
  "watchers": 16,
  "default_branch": "master",
  "organization": {
    "login": "BoostGSoC13",
    "id": 4558366,
    "avatar_url": "https://avatars.githubusercontent.com/u/4558366?",
    "gravatar_id": null,
    "url": "https://api.github.com/users/BoostGSoC13",
    "html_url": "https://github.com/BoostGSoC13",
    "followers_url": "https://api.github.com/users/BoostGSoC13/followers",
    "following_url": "https://api.github.com/users/BoostGSoC13/following{/other_user}",
    "gists_url": "https://api.github.com/users/BoostGSoC13/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/BoostGSoC13/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/BoostGSoC13/subscriptions",
    "organizations_url": "https://api.github.com/users/BoostGSoC13/orgs",
    "repos_url": "https://api.github.com/users/BoostGSoC13/repos",
    "events_url": "https://api.github.com/users/BoostGSoC13/events{/privacy}",
    "received_events_url": "https://api.github.com/users/BoostGSoC13/received_events",
    "type": "Organization",
    "site_admin": false
  },
  "network_count": 5,
  "subscribers_count": 10
}
