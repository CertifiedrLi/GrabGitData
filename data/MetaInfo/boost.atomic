{
  "id": 1805935,
  "name": "boost.atomic",
  "full_name": "brycelelbach/boost.atomic",
  "owner": {
    "login": "brycelelbach",
    "id": 398194,
    "avatar_url": "https://avatars.githubusercontent.com/u/398194?",
    "gravatar_id": "210165afae8965cd02b3acd6fa932eec",
    "url": "https://api.github.com/users/brycelelbach",
    "html_url": "https://github.com/brycelelbach",
    "followers_url": "https://api.github.com/users/brycelelbach/followers",
    "following_url": "https://api.github.com/users/brycelelbach/following{/other_user}",
    "gists_url": "https://api.github.com/users/brycelelbach/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/brycelelbach/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/brycelelbach/subscriptions",
    "organizations_url": "https://api.github.com/users/brycelelbach/orgs",
    "repos_url": "https://api.github.com/users/brycelelbach/repos",
    "events_url": "https://api.github.com/users/brycelelbach/events{/privacy}",
    "received_events_url": "https://api.github.com/users/brycelelbach/received_events",
    "type": "User",
    "site_admin": false
  },
  "private": false,
  "html_url": "https://github.com/brycelelbach/boost.atomic",
  "description": "Hartmut Kaiser and Bryce Lelbach's version of Boost.Atomic",
  "fork": false,
  "url": "https://api.github.com/repos/brycelelbach/boost.atomic",
  "forks_url": "https://api.github.com/repos/brycelelbach/boost.atomic/forks",
  "keys_url": "https://api.github.com/repos/brycelelbach/boost.atomic/keys{/key_id}",
  "collaborators_url": "https://api.github.com/repos/brycelelbach/boost.atomic/collaborators{/collaborator}",
  "teams_url": "https://api.github.com/repos/brycelelbach/boost.atomic/teams",
  "hooks_url": "https://api.github.com/repos/brycelelbach/boost.atomic/hooks",
  "issue_events_url": "https://api.github.com/repos/brycelelbach/boost.atomic/issues/events{/number}",
  "events_url": "https://api.github.com/repos/brycelelbach/boost.atomic/events",
  "assignees_url": "https://api.github.com/repos/brycelelbach/boost.atomic/assignees{/user}",
  "branches_url": "https://api.github.com/repos/brycelelbach/boost.atomic/branches{/branch}",
  "tags_url": "https://api.github.com/repos/brycelelbach/boost.atomic/tags",
  "blobs_url": "https://api.github.com/repos/brycelelbach/boost.atomic/git/blobs{/sha}",
  "git_tags_url": "https://api.github.com/repos/brycelelbach/boost.atomic/git/tags{/sha}",
  "git_refs_url": "https://api.github.com/repos/brycelelbach/boost.atomic/git/refs{/sha}",
  "trees_url": "https://api.github.com/repos/brycelelbach/boost.atomic/git/trees{/sha}",
  "statuses_url": "https://api.github.com/repos/brycelelbach/boost.atomic/statuses/{sha}",
  "languages_url": "https://api.github.com/repos/brycelelbach/boost.atomic/languages",
  "stargazers_url": "https://api.github.com/repos/brycelelbach/boost.atomic/stargazers",
  "contributors_url": "https://api.github.com/repos/brycelelbach/boost.atomic/contributors",
  "subscribers_url": "https://api.github.com/repos/brycelelbach/boost.atomic/subscribers",
  "subscription_url": "https://api.github.com/repos/brycelelbach/boost.atomic/subscription",
  "commits_url": "https://api.github.com/repos/brycelelbach/boost.atomic/commits{/sha}",
  "git_commits_url": "https://api.github.com/repos/brycelelbach/boost.atomic/git/commits{/sha}",
  "comments_url": "https://api.github.com/repos/brycelelbach/boost.atomic/comments{/number}",
  "issue_comment_url": "https://api.github.com/repos/brycelelbach/boost.atomic/issues/comments/{number}",
  "contents_url": "https://api.github.com/repos/brycelelbach/boost.atomic/contents/{+path}",
  "compare_url": "https://api.github.com/repos/brycelelbach/boost.atomic/compare/{base}...{head}",
  "merges_url": "https://api.github.com/repos/brycelelbach/boost.atomic/merges",
  "archive_url": "https://api.github.com/repos/brycelelbach/boost.atomic/{archive_format}{/ref}",
  "downloads_url": "https://api.github.com/repos/brycelelbach/boost.atomic/downloads",
  "issues_url": "https://api.github.com/repos/brycelelbach/boost.atomic/issues{/number}",
  "pulls_url": "https://api.github.com/repos/brycelelbach/boost.atomic/pulls{/number}",
  "milestones_url": "https://api.github.com/repos/brycelelbach/boost.atomic/milestones{/number}",
  "notifications_url": "https://api.github.com/repos/brycelelbach/boost.atomic/notifications{?since,all,participating}",
  "labels_url": "https://api.github.com/repos/brycelelbach/boost.atomic/labels{/name}",
  "releases_url": "https://api.github.com/repos/brycelelbach/boost.atomic/releases{/id}",
  "created_at": "2011-05-26T18:42:35Z",
  "updated_at": "2013-09-03T06:25:33Z",
  "pushed_at": "2011-05-28T18:40:34Z",
  "git_url": "git://github.com/brycelelbach/boost.atomic.git",
  "ssh_url": "git@github.com:brycelelbach/boost.atomic.git",
  "clone_url": "https://github.com/brycelelbach/boost.atomic.git",
  "svn_url": "https://github.com/brycelelbach/boost.atomic",
  "homepage": "",
  "size": 159,
  "stargazers_count": 5,
  "watchers_count": 5,
  "language": "C++",
  "has_issues": true,
  "has_downloads": true,
  "has_wiki": true,
  "forks_count": 0,
  "mirror_url": null,
  "open_issues_count": 0,
  "forks": 0,
  "open_issues": 0,
  "watchers": 5,
  "default_branch": "master",
  "network_count": 0,
  "subscribers_count": 1
}
