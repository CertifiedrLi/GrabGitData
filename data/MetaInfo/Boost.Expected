{
  "id": 9867410,
  "name": "Boost.Expected",
  "full_name": "ptal/Boost.Expected",
  "owner": {
    "login": "ptal",
    "id": 3127628,
    "avatar_url": "https://avatars.githubusercontent.com/u/3127628?",
    "gravatar_id": "8ff84b5734a780738d3fc9c74a332221",
    "url": "https://api.github.com/users/ptal",
    "html_url": "https://github.com/ptal",
    "followers_url": "https://api.github.com/users/ptal/followers",
    "following_url": "https://api.github.com/users/ptal/following{/other_user}",
    "gists_url": "https://api.github.com/users/ptal/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/ptal/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/ptal/subscriptions",
    "organizations_url": "https://api.github.com/users/ptal/orgs",
    "repos_url": "https://api.github.com/users/ptal/repos",
    "events_url": "https://api.github.com/users/ptal/events{/privacy}",
    "received_events_url": "https://api.github.com/users/ptal/received_events",
    "type": "User",
    "site_admin": false
  },
  "private": false,
  "html_url": "https://github.com/ptal/Boost.Expected",
  "description": "What did you expect?",
  "fork": false,
  "url": "https://api.github.com/repos/ptal/Boost.Expected",
  "forks_url": "https://api.github.com/repos/ptal/Boost.Expected/forks",
  "keys_url": "https://api.github.com/repos/ptal/Boost.Expected/keys{/key_id}",
  "collaborators_url": "https://api.github.com/repos/ptal/Boost.Expected/collaborators{/collaborator}",
  "teams_url": "https://api.github.com/repos/ptal/Boost.Expected/teams",
  "hooks_url": "https://api.github.com/repos/ptal/Boost.Expected/hooks",
  "issue_events_url": "https://api.github.com/repos/ptal/Boost.Expected/issues/events{/number}",
  "events_url": "https://api.github.com/repos/ptal/Boost.Expected/events",
  "assignees_url": "https://api.github.com/repos/ptal/Boost.Expected/assignees{/user}",
  "branches_url": "https://api.github.com/repos/ptal/Boost.Expected/branches{/branch}",
  "tags_url": "https://api.github.com/repos/ptal/Boost.Expected/tags",
  "blobs_url": "https://api.github.com/repos/ptal/Boost.Expected/git/blobs{/sha}",
  "git_tags_url": "https://api.github.com/repos/ptal/Boost.Expected/git/tags{/sha}",
  "git_refs_url": "https://api.github.com/repos/ptal/Boost.Expected/git/refs{/sha}",
  "trees_url": "https://api.github.com/repos/ptal/Boost.Expected/git/trees{/sha}",
  "statuses_url": "https://api.github.com/repos/ptal/Boost.Expected/statuses/{sha}",
  "languages_url": "https://api.github.com/repos/ptal/Boost.Expected/languages",
  "stargazers_url": "https://api.github.com/repos/ptal/Boost.Expected/stargazers",
  "contributors_url": "https://api.github.com/repos/ptal/Boost.Expected/contributors",
  "subscribers_url": "https://api.github.com/repos/ptal/Boost.Expected/subscribers",
  "subscription_url": "https://api.github.com/repos/ptal/Boost.Expected/subscription",
  "commits_url": "https://api.github.com/repos/ptal/Boost.Expected/commits{/sha}",
  "git_commits_url": "https://api.github.com/repos/ptal/Boost.Expected/git/commits{/sha}",
  "comments_url": "https://api.github.com/repos/ptal/Boost.Expected/comments{/number}",
  "issue_comment_url": "https://api.github.com/repos/ptal/Boost.Expected/issues/comments/{number}",
  "contents_url": "https://api.github.com/repos/ptal/Boost.Expected/contents/{+path}",
  "compare_url": "https://api.github.com/repos/ptal/Boost.Expected/compare/{base}...{head}",
  "merges_url": "https://api.github.com/repos/ptal/Boost.Expected/merges",
  "archive_url": "https://api.github.com/repos/ptal/Boost.Expected/{archive_format}{/ref}",
  "downloads_url": "https://api.github.com/repos/ptal/Boost.Expected/downloads",
  "issues_url": "https://api.github.com/repos/ptal/Boost.Expected/issues{/number}",
  "pulls_url": "https://api.github.com/repos/ptal/Boost.Expected/pulls{/number}",
  "milestones_url": "https://api.github.com/repos/ptal/Boost.Expected/milestones{/number}",
  "notifications_url": "https://api.github.com/repos/ptal/Boost.Expected/notifications{?since,all,participating}",
  "labels_url": "https://api.github.com/repos/ptal/Boost.Expected/labels{/name}",
  "releases_url": "https://api.github.com/repos/ptal/Boost.Expected/releases{/id}",
  "created_at": "2013-05-05T11:01:29Z",
  "updated_at": "2014-06-23T23:12:48Z",
  "pushed_at": "2014-06-23T23:12:48Z",
  "git_url": "git://github.com/ptal/Boost.Expected.git",
  "ssh_url": "git@github.com:ptal/Boost.Expected.git",
  "clone_url": "https://github.com/ptal/Boost.Expected.git",
  "svn_url": "https://github.com/ptal/Boost.Expected",
  "homepage": "",
  "size": 2210,
  "stargazers_count": 10,
  "watchers_count": 10,
  "language": "C++",
  "has_issues": true,
  "has_downloads": true,
  "has_wiki": true,
  "forks_count": 3,
  "mirror_url": null,
  "open_issues_count": 10,
  "forks": 3,
  "open_issues": 10,
  "watchers": 10,
  "default_branch": "master",
  "network_count": 3,
  "subscribers_count": 5
}
